deploy/roles/eod/tasks/main.yaml

```diff
@@ -49,10 +49,27 @@
   tags:
     - eod
 
+- name: Create admin group
+  group:
+    name: "{{ restricted_group_name }}"
+    state: present
+
```

Comment:
@stirlingv The common role is applied in the `hosts: research_environment` block at the top of the playbook, which applies to all servers in the RE, so you can safely remove this block, thanks!